src/AutoUI/Actions/Create.tsx

```diff
@@ -55,42 +57,46 @@ export const Create = <T extends AutoUIBaseResource<T>>({
 		disabledReasonsByAction[action.title];
 
 	return (
-		<Box>
-			<Button
-				data-action={`create-${model.resource}`}
-				onClick={() =>
-					onActionTriggered({
-						action,
-						schema: autoUIJsonSchemaPick(
-							model.schema,
-							model.permissions.create,
-						),
-					})
-				}
-				icon={<FontAwesomeIcon icon={faMagic} />}
-				tooltip={
-					typeof disabledReason === 'string' ? disabledReason : undefined
-				}
-				disabled={!!disabledReason}
-				primary
+		<Box display="flex">
+			<Tooltip
+				title={typeof disabledReason === 'string' ? disabledReason : undefined}
 			>
-				<ActionContent<T>
-					getDisabledReason={action.isDisabled}
-					affectedEntries={undefined}
-					checkedState={undefined}
-					onDisabledReady={(result) => {
-						setDisabledReasonsByAction((disabledReasonsState) => ({
-							...disabledReasonsState,
-							[action.title]: result,
-						}));
-					}}
+				<Button
+					data-action={`create-${model.resource}`}
+					variant="contained"
+					onClick={() =>
+						onActionTriggered({
+							action,
+							schema: autoUIJsonSchemaPick(
+								model.schema,
+								model.permissions.create,
+							),
+						})
+					}
+					startIcon={<AutoFixNormal />}
```

Comment:
If we are officially updating icons to MUI ones, we should add a task to our projects/improvements to try to drop fontawesome in favor of MUI (if we succeed that may also save us money since we pay for pro fontawesome)